### PR TITLE
Added Travis CI

### PR DESCRIPTION
**Pull request by twolfson**
We have decided to add Travis CI in #15. In this PR:

- Fixed up `.travis.yml` to test against multiple Pythons
- Added `test.sh` (`tox.ini` is yet another tool for everyone to learn otherwise =/)
- Fixed up lint error
- Added Travis CI badge

/cc @zheller

### CODE REVIEW

File: flake8_quotes.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def __init__(self, tree, filename='(none)', builtins=None):
     def add_options(cls, parser):
         parser.add_option('--quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
-        parser.config_options.append("quotes")
+        parser.config_options.append('quotes')
```
Comment (zheller) [inline flake8_quotes.py]: haha! how fake8-quotes uses flake8-quotes to lint flake8-quotes

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```
Comment (zheller) [inline setup.cfg]: I'm a 79 man myself, but do as you will

**Comment by zheller** [comment]
merge at will

File: setup.cfg (hunk 2 above)
Comment (twolfson) [inline setup.cfg]: There are a few lines that go over 80 characters already =/

**Comment by twolfson** [comment]
woot :shipit:

**Comment by twolfson** [comment]
This has been merged/released in `0.2.1`.

**Pull request merged**